lib/rucio/client/ruleclient.py

```diff
@@ -143,6 +143,25 @@ def reduce_replication_rule(self, rule_id, copies, exclude_expression=None):
             exc_cls, exc_msg = self._get_exception(headers=r.headers, status_code=r.status_code, data=r.content)
             raise exc_cls(exc_msg)
 
+    def move_replication_rule(self, rule_id, rse_expression):
+        """
+        Move a replication rule to another RSE and, once done, delete the original one.
+
+        :param rule_id:             Rule to be moved.
+        :param rse_expression:      RSE expression of the new rule.
+        :raises:                    RuleNotFound, RuleReplaceFailed
+        """
+
+        path = self.RULE_BASEURL + '/' + rule_id + '/move'
+        url = build_url(choice(self.list_hosts), path=path)
+        data = dumps({'rule_id': rule_id, 'rse_expression': rse_expression})
+        r = self._send_request(url, type='POST', data=data)
+        if r.status_code == codes.created:
+            return loads(r.text)
+        else:
```

Comment:
useless else:

Reply:
True, but I went for consistency here, as it is like this in all clients.

Reply:
yes and I always try to remove it at few places. Not blocking anyway

Reply:
True. I'll change it.